ui/index.js

```diff
@@ -19,9 +19,20 @@ class Dashboard extends Component {
 		})
 	}
 
+  handleShiftTab = () => {
+    const { processes } = this.props
+    const newIndex = --this.state.active
```

Comment:
Can we use a single handler? I wonder if we can distinguise using the event object
`screen.key(['tab', 'S-tab'], this.handleTab)`

Also indent is mess, you can hit a reformat to the whole file with 4 space indents.

Reply:
Yes, sorry for the indents, I will fix it. Yes, I just found that we can use a single handler. I will refactor this.